README.md

```diff
@@ -151,6 +151,36 @@ type NftMetadata = {
 
 See the implementation of the [Ethers](https://github.com/spectrexyz/use-nft/blob/38bd803f20e778b9bb684d682c194a812a94a05c/src/fetchers/ethers/index.tsx#L12-L42) and [Ethereum](https://github.com/spectrexyz/use-nft/blob/38bd803f20e778b9bb684d682c194a812a94a05c/src/fetchers/ethereum/index.tsx#L12-L42) fetchers for more details.
 
+## Angular support
+
+To add support for this library, first install `core-js`
+
+```
+npm i -s core-js
+```
+
+then add the following line to `src/polyfills.ts`
+
+```
+import 'core-js/features/promise';
+```
```

Comment:
The library uses `Promise.Any`, which is part of ES2021, so it's not even released yet. That's why I had to make a workaround. I will remove this part but probably I will also create an issue where I explain how to get it working in case an Angular dev encounters the same problem as I do

Reply:
Oh I didn’t think of that! I’m going to replace the `Promise.any()` calls with a JS implementation to avoid this issue. :+1:

Reply:
Done in https://github.com/spectrexyz/use-nft/commit/5bb26c9a29cf19a047faa8ee7b66faa347a9cdf4